common/src/main/java/com/hortonworks/streamline/common/FileWatcher.java

```diff
@@ -79,7 +79,7 @@ public boolean processEvents() {
             WatchEvent<Path> ev = (WatchEvent<Path>) event;
             Path name = ev.context();
             Path child = dir.resolve(name);
-            LOG.info(String.format("%s: %s\n", event.kind().name(), child));
+            LOG.info(String.format("%s: %s%n", event.kind().name(), child));
```

Comment:
👍 Thanks. Since there were too many issues I was blindly following the recommendations and didn't dig deeper.